hoomd/md/update.py

```diff
@@ -130,8 +131,8 @@ def __init__(self, group, r=None, rx=None, ry=None, rz=None, P=(0, 0, 0)):
         self.metadata_fields = ['group', 'P', 'rx', 'ry', 'rz']
 
 
-class mueller_plathe_flow:
-    r"""Updater for a shear flow via an algorithm published by Mueller Plathe.
+class ReversePerturbationFlow(Updater):
+    """Reverse Perturbation (Müller-Plathe) method to establish shear flow.
```

Comment:
Why aren't we using a raw string here?

Reply:
The formula in the description of flow_target does not render if a raw string is used